Livecheckables/net-snmp.rb

```diff
@@ -0,0 +1,4 @@
+class NetSnmp
+  livecheck :url   => "https://sourceforge.net/projects/net-snmp/",
+            :regex => %r{/net-snmp-([0-9\.]+)\.t}
```

Comment:
```suggestion
            :regex => %r{url=.+?/net-snmp-v?(\d+(?:\.\d+)+)\.t}
```

We can reduce the number of duplicate matches by restricting the regex to only match within the `url` attribute.

As explained on previous occasions, we want to avoid using the old `([0-9\.]+)` regex to match numeric versions, since it can lead to problematic situations. As recommended before, just use some version of the standard regex for numeric versions (i.e., `v?(\d+(?:\.\d+)+)`).

Reply:
👍